### ISSUE TITLE
Air: eliminate `--rebuild-registry` feature after #663 is merged into Release and RC

### ISSUE THREAD

**Issue by host6** [voedger/voedger#756]
## Problem

**Now, in current architecture**
`sys.RecordsRegistryProjector` could be run in async mode according to #663

- Problem (warm): async mode is a dept. Need to eliminate it after #663 is merged into RC and Release

**Describe the solution you'd like**
- voedger: eliminate ability to create the projector in async mode
- airs-bp3: eliminate `--rebuild-registry` cmd line arg
